tests/files/test_route_output_files.py

```diff
@@ -94,26 +94,12 @@ def test_get_last_output_files(self):
         output_files = self.get(
             "/data/entities/%s/output-files/last-revisions" % self.asset.id
         )
-        self.assertEqual(
-            output_files[str(self.geometry_id)]["main"],
-            self.output_file_geometry.serialize()
-        )
-        self.assertEqual(
-            output_files[str(self.cache_type_id)]["main"],
-            self.output_file_cache.serialize()
-        )
-        self.assertEqual(
-            output_files[str(self.tx_type_id)]["main"],
-            self.output_file_texture.serialize()
-        )
-        self.assertEqual(
-            output_files[str(self.render_id)]["main"],
-            self.output_file_render_1.serialize()
-        )
-        self.assertEqual(
-            output_files[str(self.render_id)]["variant-1"],
-            self.output_file_render_2.serialize()
-        )
+
+        assert(self.output_file_geometry.serialize() in output_files)
+        assert(self.output_file_cache.serialize() in output_files)
+        assert(self.output_file_texture.serialize() in output_files)
+        assert(self.output_file_render_1.serialize() in output_files)
+        assert(self.output_file_render_2.serialize() in output_files)
```

Comment:
Can you use `self.assertIn(a, b)` instead?

Reply:
done